src/github.com/getlantern/flashlight/pac.go

```diff
@@ -41,16 +44,33 @@ func setUpPacTool() error {
 	return nil
 }
 
+func setProxyAddr(addr string) {
```

Comment:
Yeah you are right, my brain stuck. A PAC file can be changed dynamically. Will do it right away.